src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeService.kt

```diff
@@ -658,7 +658,7 @@ class NodeService(
         .flatMap { email }
         .map {
           MyBankAdditionalPaymentInformationsDto().apply {
-            this.transactionId = npgTransactionGatewayAuthorizationData.operationId
+            this.transactionId = npgTransactionGatewayAuthorizationData.paymentEndToEndId
```

Comment:
```suggestion
            this.transactionId = authCompleted.transactionId.value()
```

here we have to use eCommerce transaction id as per task specification, right?

Reply:
Yep